### PR TITLE
Add installation requirements for Python and error to FAQ.

### PR DESCRIPTION
**Pull request by anton-bystrov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
 This repository serves the purpose of creating and/or updating the feature builds for PMM Server and PMM Client. It is auxiliary
 to our build system managed by Jenkins as it helps pull the right branches from different repositories that PMM [consists of](https://github.com/percona/pmm).
 
+# Installation dependencies
+
+If you build with Python's script first of all install requirements:
```
Comment (nikita-b) [inline README.md]:
```suggestion
If you build with Python's script then you need install requirements:
```

**Review by nikita-b — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2156-6034a22
client docker - perconalab/pmm-client-fb:PR-2156-6034a22
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2156-6034a22.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2156-6034a22&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2156-6034a22.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2156-d32dd67
client docker - perconalab/pmm-client-fb:PR-2156-d32dd67
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2156-d32dd67.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2156-d32dd67&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2156-d32dd67.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/8563/

**Pull request merged**